pkgs/development/python-modules/dns-lexicon/default.nix

```diff
@@ -0,0 +1,42 @@
+{ buildPythonPackage
+, fetchPypi
+, poetry-core
+, beautifulsoup4
+, cryptography
+, importlib-metadata
+, pyyaml
+, requests
+, tldextract
+, lib
+}:
+
+buildPythonPackage rec {
+  pname = "dns_lexicon";
+  version = "3.14.1";
+  format = "pyproject";
+
+  nativeBuildInputs = [ poetry-core ];
+
+  propagatedBuildInputs = [
+    beautifulsoup4
+    cryptography
+    importlib-metadata
+    pyyaml
+    requests
+    tldextract
+  ];
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-+h1xt5vH49BWsKqxoRFg1X18Uk6qUPQwNVLWr/L/5RQ=";
+  };
+
```

Comment:
Could you please enable checkPhase and pythonImportsCheck?
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#testing-python-packages-testing-python-packages

Reply:
Done, thank you!